src/main/java/ua/ithillet/lesson4/ArrayValueCalculator.java

```diff
@@ -0,0 +1,28 @@
+package ua.ithillet.lesson4;
+
+import static ua.ithillet.lesson4.Main.LENGHT_OF_ARRAY;
+
+public class ArrayValueCalculator {
+
+    public int doCalc(String[][] array) {
+        int sum = 0;
+        if (LENGHT_OF_ARRAY < array.length) {
```

Comment:
замість двох перевірок, я би робив одну (друга, на 13-й строкі)
```suggestion
        if (LENGHT_OF_ARRAY != array.length) {
```